src/app/pages/profile/personal-information/personal-information.component.ts

```diff
@@ -22,14 +22,14 @@ export class PersonalInformationComponent implements OnInit, OnDestroy {
   };
   profileForm: FormGroup;
   firstName = new FormControl(
-    { value: '', disabled: true },
+    { value: "", disabled: true },
```

Comment:
hey @Jaman-dedy, we should use single quotes in ts files.

Reply:
I worked on that @CryceTruly , thanks